src/app/heading-banner/heading-banner.component.ts

```diff
@@ -12,14 +12,17 @@ import { firstValueFrom } from 'rxjs';
  *
  */
 @Component({
-  selector: 'app-meta-campaign-banner',
+  selector: 'app-heading-banner',
   imports: [AsyncPipe, OptimisedImagePipe, NgStyle],
-  templateUrl: './meta-campaign-banner.component.html',
-  styleUrl: './meta-campaign-banner.component.scss',
+  templateUrl: './heading-banner.component.html',
+  styleUrl: './heading-banner.component.scss',
 })
-export class MetaCampaignBannerComponent {
+export class HeadingBanner {
   imageService = inject(ImageService);
   logo = input<{ url: string; alt: string | undefined } | undefined>();
+  /** Optional slightly smaller text to appear above the main title */
+  slug = input('');
```

Comment:
I don't really like the name slug, I would expect a slug to be a URL component, but slug matches what we're using on the old component and I can't immediately think of anything better.